### DISCUSSION TITLE
Demo/state of Pinebook Pro

### DISCUSSION THREAD

**Discussion by cassidyjames** [elementary/streams#27] [Ideas]
It would be cool to show a kind of hands on with the Pinebook Pro, and stream the real-world performance of it running elementary OS. Maybe going over some of the enablement efforts.

**Comment by cassidyjames** [comment]
Could also be expanded to state of elementary OS on ARM with a Raspberry Pi 400 demo.

**Reply by derikatwork** [reply]
As someone who is getting into the ideation phase of product creation around the Pi and Pine64 boards, I would be very interested in seeing the progress being made.
